Source/Models/APIClient.swift

```diff
@@ -4,46 +4,35 @@ import Result
 
 public struct APIClient {
   private let requestPerformer: RequestPerformer
+  public let deserializer: Deserializer
```

Comment:
After discussion in slack, I think @gfontenot is right here, I'll make both private and remove them from the `Client` protocol.